astropy/coordinates/angle_formats.py

```diff
@@ -309,7 +309,7 @@ def parse(self, angle, unit, debug=False):
             )
         except ValueError as e:
             if str(e):
-                raise ValueError(f"{str(e)} in angle {angle!r}") from e
+                raise ValueError(f"{e!s} in angle {angle!r}") from e
```

Comment:
Could this be simplified even more? There's a few more cases like this elsewhere.
```suggestion
                raise ValueError(f"{e} in angle {angle!r}") from e
```

Reply:
Done.